rosboard/html/js/viewers/meta/Space3DViewer.js

```diff
@@ -90,6 +90,56 @@ class Space3DViewer extends Viewer {
       that.updatePerspective();
     }
 
+    var touchRegion = ZingTouch.Region(this.gl.canvas)
+
+    // One finger drag to rotate
+    var rotateGesture = new ZingTouch.Pan({
+      numInputs: 1
+    })
+    touchRegion.register('rotateGesture', rotateGesture)
+
+    touchRegion.bind(this.gl.canvas, 'rotateGesture', (e) => {
+      console.log(e.detail)
+      if (e.detail.data[0]) {
+        const change = e.detail.data[0].change;
+        if (Math.abs(change.x) > 100 || Math.abs(change.y) > 100) return;
```

Comment:
Where did all of these hard-coded numbers come from?  Manual tuning?

Reply:
Yes, they adjust the sensitivity of the gestures.

Reply:
Can these be set via a `Space3DViewerSettings` class?

Reply:
Please create a follow-on ticket to address.